src/App.css

```
@@ -1,13 +1,96 @@
+
+html.app-background-visible .tab-page-wrapper {
+  background: transparent !important;
+}
+
 html {
   width: 100%;
   overflow-x: hidden;
 }
 
 body, .App {
   font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)